File: search_app.py
import re

with open('app.js', 'r', encoding='utf-8') as f:
    lines = f.readlines()

queries = ['openOperationLogModal', 'addRow', 'openClientModal', 'handleModalSubmit', 'setupRouting', 'routing-tl-pills', 'sanitation-report', 'reports-tab', 'report-type', 'baitStationsChart', 'generateReport', 'UoM']

for query in queries:
    print(f"=== Matches for '{query}' ===")
    for idx, line in enumerate(lines):
        if query in line:
            print(f"{idx+1}: {line.strip()[:100]}")

File: search_app.js
const fs = require('fs');

const content = fs.readFileSync('app.js', 'utf8');
const lines = content.split('\n');

const queries = [
  'openOperationLogModal',
  'addRow',
  'openClientModal',
  'handleModalSubmit',
  'setupRouting',
  'routing-tl-pills',
  'sanitation-report',
  'reports-tab',
  'report-type',
  'baitStationsChart',
  'generateReport',
  'UoM'
];

queries.forEach(query => {
  console.log(`=== Matches for '${query}' ===`);
  lines.forEach((line, idx) => {
    if (line.includes(query)) {
      console.log(`${idx + 1}: ${line.trim().substring(0, 100)}`);
    }
  });
});
